homeassistant/components/speedtestdotnet/__init__.py

```diff
@@ -171,15 +171,15 @@ def update_data(self):
                 "Executing speedtest.net speed test with server_id: %s",
```

Comment:
Please move the debug log down too.

Reply:
Sorry, not something I added, so not sure what it is about.

Reply:
Ahh, I see what you mean, sorry.